src/test/java/valueobject/GoogleCloudForm.java

```diff
@@ -0,0 +1,81 @@
+package valueobject;
+
+import excpetion.CommittedUsageException;
+import excpetion.GPUNumberException;
+import excpetion.LocalSSDNumberException;
+
+import java.util.Objects;
+
+public class GoogleCloudForm {
+
+	private final int gpuNumber;
+	private final String gpuTypeValue;
+	private final int localSSDNumber;
+	private final String datacenterLocationValue;
+	private final int committedUsageNumber;
+
+	public GoogleCloudForm(int gpuNumber, String gpuTypeValue, int localSSDNumber
+			, String datacenterLocationValue, int committedUsageNumber) {
+		this.gpuNumber = gpuNumber;
+		this.gpuTypeValue = gpuTypeValue;
+		this.localSSDNumber = localSSDNumber;
+		this.datacenterLocationValue = datacenterLocationValue;
+		this.committedUsageNumber = committedUsageNumber;
+	}
+
+	public int getGpuNumber() {
+		if (gpuNumber <= 0) {
+			try {
```

Comment:
This try/catch construction looks quite weird. Why do we need to throw an exception and catch it right away?